### ISSUE TITLE
Hyphenation - incorrect corrections

### ISSUE THREAD

**Issue by dylan-chong** [redpen-cc/redpen#826]
For text:

    It would be able to work.

This error occurs:

![screen shot 2018-03-30 at 12 58 41 pm](https://user-images.githubusercontent.com/17482349/38118860-1b931194-341a-11e8-8963-fa5ce50b69b2.png)

However, I don't believe this is a correct suggestion.

**Comment by dylan-chong** [comment]
Another example: `Information has been copied below # redpen suggests 'has-been' should be hyphenated, but i have never seen those two words hyphenated before`

**Comment by nicolaiskogheim** [comment]
```
Thank you for your interest.
```

`Hyphenation: This phrase should be hyphenated (ie: "Thank-you").`

Thank-you, redpen. 😁

**Comment by dylan-chong** [comment]
@nicolaiskogheim  No, not always https://ell.stackexchange.com/questions/58894/should-i-hyphenate-thank-you

This is the reason for incorrect hyphenation. Redpen assumes that these words are nouns which should be hyphenated, when they often are verbs, which should not be.

**Comment by takahi-i** [comment]
Sorry for the late response 🙇 It looks like we need to apply part of speech tagger to handle this problem 🤔

**Comment by dylan-chong** [comment]
After some consideration, I believe the best solution would be to disable hyphenation checking for now. 

I think that uses of the words that should be hyphenated are quite rare, and uses of the words that should not be hyphenated are very common. Therefore there would be many warnings that one has to ignore, which can lead to accidentally ignoring real warnings. It is also confusing to be asked to use red pen to check grammar, then be asked to ignore some warnings.

Given your response above @takahi-i , it may be tricky and time consuming to solve this problem.
